### ISSUE TITLE
Example with Smart Contract function that has multiple return calls 

### ISSUE THREAD

**Issue by benjaminmbrown** [Horyus/vortex#19]
Hi @mortimr 

How can we ingest and show responses on the FE from Smart Contract functions that have multiple return calls?

```
 function getBountyStates(uint _bountyId) public view returns (bool,bool, bool){
        bool init = bounties[_bountyId].taken;
        bool complete = bounties[_bountyId].complete;
        bool verified = bounties[_bountyId].verified;
        return (init, complete, verified);
    }

```

**Comment by mortimr** [comment]
Hi @benjaminmbrown,

Having 1, 2, 3 or even more return variables does not change anything in how you use Vortex. All I'm doing is wrap the call you would be doing to web3 inside Vortex, cache the return value and return it to you.
I just tried doing it in my example by simply editing one of my method to return two uints instead of 1 and I recovered the result properly.

Have you tried to [call the method like any other constant method](https://github.com/Horyus/ticket721-webapp/blob/055af52d2e7533e6322cc5c1319ebd28758191cd/src/components/ticket_showcase/index.js#L102-L103) ?
